src/ui/components/PasswordModule/PasswordModule.tsx

```diff
@@ -118,6 +118,14 @@ const PasswordModule = forwardRef<PasswordModuleRef, PasswordModuleProps>(
               content: { value: hintValue },
             })
           );
+        } else {
+          try {
+            await Agent.agent.basicStorage.deleteById(
+              MiscRecordId.OP_PASS_HINT
+            );
+          } catch (e) {
+            // TODO: handle error
```

Comment:
As I mentioned in the last PR, moving forward we should avoid leaving empty catch. Please handle this error in the same way you did last time.